lib/postgrex/pgpass.ex

```diff
@@ -47,17 +36,45 @@ defmodule Postgrex.Pgpass do
     |> Enum.map(&(String.split(&1, ":")))
   end
 
-  defp readable_pgpass_file? do
-    with {:ok, stat} <- File.stat(pgpass_path()),
+  defp readable_pgpass_file?(nil) do
+    # when the pgpass is not passed via the :passfile option, simply ignore and move on
+    with default_path <- pgpass_path(),
+         {:ok, stat} <- File.stat(default_path),
          0o0600 <- stat.mode &&& 0o0777,
-    do: true,
+    do: {:ok, default_path},
```

Comment:
psql only warns, then won't use the file. Which means you can fix it later - a raise here would obligate you to fix the problem. Should we warn like psql?